modules/conflict_resolver/static/schema.yml

```diff
@@ -0,0 +1,260 @@
+openapi: 3.0.1
+info:
+  title: LORIS - Conflict Resolver Module
+  description: Endpoints available for the Conflict Resolver module
+  contact:
+    name: LORIS Development Team
+    url: 'https://github.com/aces/loris'
+  license:
+    name: 'GNU Public License, Version 3'
+    url: 'https://opensource.org/licenses/GPL-3.0'
+  version: 1.0.0
+servers:
+  - url: /
+paths:
+  /conflict_resolver:
+    get:
+      tags:
+        - Conflict Resolver
+      summary: Generates the html page for this module including the React component. 
+      responses:
+        '200':
+          description: The html page with menu, breadcrumbs and the React component that interact with the backend for data requests and submission.
+          content:
+            text/html:
+              schema:
+                type: string
+
+  /conflict_resolver/unresolved:
+    get:
+      tags:
+        - Conflict Resolver
+      summary: Get all unresolved conflicts
+      responses:
+        '200':
+          description: An object containing the data and fieldOptions where data are the conflicts and fieldOptions are the options for the available filters.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnresolvedMenuFilterResponse'
+    post:
+      tags:
+        - Conflict Resolver
+      summary: Fix a given conflict by providing the correct answer.
+      description: user to add to the system
+      requestBody:
+        description: Optional description in *Markdown*
```

Comment:
This doesn't look like it's correct.. as far as I know the conflict resolver doesn't do anything with markdown?

Reply:
You are right. This looks like copy-pasta...